diagnostic.md

```diff
@@ -7,36 +7,39 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:laurpaik/git-basics-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!--enter the new repository and switch the branch:
+`git checkout -b response`-->
```

Comment:
You do want to switch branches to make your changes in, but the branch needs to created first.